### DISCUSSION TITLE
兴，百姓苦；亡，百姓苦

### DISCUSSION THREAD

**Discussion by HunterTOne** [WeNeedHome/SummaryOfLoanSuspension#594] [General]
规则就是用来限制普通人的，制定规则的那帮人，可以用无数方法来钻空子、漏洞；

**Comment by Azheng197** [comment]
一人之心，千万人之心也。秦爱纷奢，人亦念其家；奈何取之尽锱铢，用之如泥沙？使负栋之柱，多于南亩之农夫；架梁之椽，多于机上之工女；钉头磷磷，多于在庾之粟粒；瓦缝参差，多于周身之帛缕；直栏横槛，多于九土之城郭；管弦呕哑，多于市人之言语。使天下之人，不敢言而敢怒；
